docs/pipeline.rst

```diff
@@ -44,13 +44,23 @@ and the full suite of headless and UI tests are then run against
 Firefox on Windows 10 using `Sauce Labs`_. This is handled by the pipeline, and is subject
 to change according to the settings in the `master.yml file`_ in the repository.
 
+Push to stage branch
+~~~~~~~~~~~~~~~~~~~~~
+
+Whenever a change is pushed to the stage branch, a production docker image is built and
+published, and the full suite of UI tests is run against Chrome and Internet Explorer on
+Windows 10, and the sanity suite is run against older versions of Internet Explorer (currently IE6 & IE7).
```

Comment:
Sanity tests are now run against IE8.

Reply:
now IE9 as of https://github.com/mozilla/bedrock/pull/6696